clang/tools/cir-opt/cir-opt.cpp

```diff
@@ -0,0 +1,46 @@
+//===- cir-opt.cpp - CIR optimization and analysis driver -----*- C++ -*-===//
```

Comment:
I think the LLVM coding rules changed recently about these lines which are no longer required as Emacs made some progress.

Reply:
Thanks, I wasn't aware of that. I'll update this. I see that I also missed the file header on one of the other files I'm adding.